src/main/java/com/gluonhq/substrate/model/ClassPath.java

```diff
@@ -0,0 +1,102 @@
+package com.gluonhq.substrate.model;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Represents the classpath
+ * Consolidates all classpath manipulations
+ */
+public class ClassPath {
+
+    private final String classPath;
+
+    /**
+     * Creates the class path
+     * @param classPath standard java classpath, usually delimited with "pathSeparator"
```

Comment:
usually? Does this mean it is not required, or do you mean it can be a single path?

Reply:
Single path.. usually is the wrong word I guess

Reply:
Done